WaltonSustainabilityCentre-Requirements.txt

```diff
@@ -1,21 +1,14 @@
-﻿REQ-ID:
-Description:
-Type:
-Rationale:
-Originator:
-Comments: [M814-ModuleChair] Copy this template below to add new requirements.
-
-REQ-ID: 001
+﻿REQ-ID: 001
 Description: The product shall record workshop information.
 Type: Functional
-Rationale: The information for each workshop must be available to the system users.
+Rationale: The information for each workshop must be available to system users, enabling staff to deliver efficient customer service.
 Originator: Karen, Centre Administrator
 Comments: Filippo Sassi
 
 REQ-ID: 002
-Description: The product shall update existing workshops information.
+Description: The product shall accept updates to existing workshop information.
```

Comment:
An internal business event could be a resource becoming unavailable. Say, for example, a classroom assigned to a workshop is flooded and so unavailable for the workshop.  An employee should be able to note the duration that resource is unavailable.  Our system should track internal resources (there isn't stated FR at the moment but this isn't a complete specification).  The WIMS should identify that a resource which was allocated to a workshop/event is now unavailable and generate the business internally.

I think both versions work but what might be needed is glossary to explain what we mean by update. I.e., is update automatic, or do it require user interaction, or both?  I think that is out of scope for this assignment and so it's fit for purpose for the moment (in my opinion of course!)